helm_lib/charts/deckhouse_lib_helm/templates/_module_security_context.tpl

```diff
@@ -69,17 +69,6 @@ securityContext:
   allowPrivilegeEscalation: false
 {{- end }}
 
-{{- /* Usage: {{ include "helm_lib_module_container_security_context_not_allow_privilege_escalation_with_selinux" . }} */ -}}
```

Comment:
1. Why did we delete this.
2. Do not edit helm_lib [directly](https://github.com/deckhouse/deckhouse/tree/main/helm_lib#helm_lib)